src/style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 300px;
+  width: 300px;
+  color: white;
+  background-color: rgb(255, 0, 0);
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  padding: 0;
+  margin: 0;
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 450px),
```

Comment:
See checklist item 2.

You should have no more than 2 media queries.

Reply:
Okay thanx have removed them now

Reply:
BACKSTOP ERROR: Reference file not found C:\Users\User\mate_projetcs\layout_snake\backstop_data\bitmaps_reference\Entire_document_450.png

I get this error when I run the test. The test images show but the reference images don't